doc/releases.md

```diff
@@ -218,6 +217,8 @@ doc/api/*.md`.
 
 *Note*: `$VERSION` should be prefixed with a `v`
 
+_Note_: `$VERSION` should be prefixed with a `v`.
```

Comment:
The change from `*` to `_` makes this inconsistent with the rest of our docs. Please change it back.

Although... even better perhaps, remove `Note:` entirely from the line.

Reply:
Oh...looks like you added a line rather than editing the existing line. So the line is there twice.

Reply:
thats my bad, i will change it back. thank you